conftest.py

```diff
@@ -14,6 +14,7 @@ def pytest_configure(config):
     PYTEST_HEADER_MODULES['pyyaml'] = 'yaml'
     PYTEST_HEADER_MODULES['scikit-image'] = 'skimage'
     PYTEST_HEADER_MODULES['specutils'] = 'specutils'
+    PYTEST_HEADER_MODULES['specreduce'] = 'specreduce'
```

Comment:
technically don't use these yet since there aren't too many API methods exposed in specreduce docs yet, but will be good to have in place since specreduce is now a dependency

Reply:
It is already a required dependency, so probably good to show version in test header.

https://github.com/spacetelescope/jdaviz/blob/70a12c3f0c56cbead9f35b606302b73fb0660fd8/setup.cfg#L37

Reply:
You also need to change the same thing in `jdaviz/conftest.py`.

Reply:
✅  added to `jdaviz/conftest.py`, thanks!